src/mixins/RunMixin.js

```diff
@@ -0,0 +1,103 @@
+import Api from '@/mixins/Api'
+import handlePromise from '@/api/PromiseHelper'
+
+export default {
+  name: 'RunMixin',
+  mixins: [Api],
+  data () {
+    return {
+      message: ''
+    }
+  },
+  methods: {
+    async getRun (id) {
+      let promise = await this.runsRequest.find(id)
```

Comment:
Do these promises still need the await? Some have it some don't.

Reply:
Ill have a look into it :)